### DISCUSSION TITLE
ios missing glyphs chars

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3687] [Mobile] [answered]
iphone 7 ios 14.3

| ios | android |
| - | - |
| ![image](https://user-images.githubusercontent.com/39104740/113487699-3a714480-94c2-11eb-97a2-f182b2d4acae.png) | ![image](https://user-images.githubusercontent.com/39104740/113487741-6b517980-94c2-11eb-8eea-0dbdd7f5d901.png) |

**Comment by brianlovin** [comment, marked as answer]
Thanks for the report! I've opened a bug issue for this.
